include/vara/Feature/FeatureModelParser.h

```diff
@@ -90,9 +90,16 @@ class FeatureModelSxfmParser : public FeatureModelParser {
   std::unique_ptr<xmlDoc, void (*)(xmlDocPtr)> parseDoc();
   bool parseVm(xmlNode *Node);
 
-  static bool parseFeatureTree(xmlChar *FeatureTree);
+  bool parseFeatureTree(xmlChar *FeatureTree);
   static bool parseConstraints(xmlChar *Constraints);
 
+  static int countOccurrences(const string& Target, const string& StringToSearch);
+  static std::optional<std::tuple<string, string>> extractCardinality(const string& StringToExtractFrom);
```

Comment:
why are the cardinalities represented as strings instead of ints, or Integer?

Reply:
Because there is also the '*' sign for the wildcard. Other than that, an integer would suffice.

Reply:
You could use INT_MAX for '*'

Reply:
I would rather prefer '-1' because '-1' as indentation is not a possible value ;-)
Or, because I have no other negative values, I could also use an unsigned int and use INT_MAX for '*'.